src/configuration/evars.md

```diff
@@ -285,6 +285,28 @@ If you have set up any [flagzones](http://book.rada.re/basic_commands/flags.html
 
 Show UTF-8 characters instead of ANSI.
 
+### scr.utf8.curvy: `bool`
+
+`scr.utf8.curvy` lets you display curved corners in places like function graphs. This variable requires `scr.utf8` to be turned on.
+
+```
+[0x0000415c]> pd 4
+│           0x0000415c      cmp   eax, 2
+│       ┌─< 0x0000415f      je    0x5001
+│       │   0x00004165      cmp   eax, 3
+│      ┌──< 0x00004168      jne   0x4348
+[0x0000415c]> e scr.utf8.curvy=1
+[0x0000415c]> pd 4
+│           0x0000415c      cmp   eax, 2
+│       ╭─< 0x0000415f      je    0x5001
+│       │   0x00004165      cmp   eax, 3
+│      ╭──< 0x00004168      jne   0x4348
+```
+
+### scr.wheel: `bool`
+
+This variable lets you toggle the mouse wheel in visual mode. Turning this variable off will be useful when you want to use your mouse to right click and copy something from the Visual mode.
```

Comment:
I would add the word "Scroll" in there to explain that the wheel can use to scroll. Also, I wonder how did you reach the conclusion about copying and right click. Should it use the mouse's wheel?

Reply:
I actually didn't know about the scroll part. I'll add it, now.
In the place of the copy and paste part, I'll just say that it lets you enable/disable mouse, then?

Reply:
```suggestion
This variable lets you enable the mouse in Visual mode. Turning this variable on will be useful when you want to use your mouse to scroll through your disassembly or copy something in the Visual mode.
```

Reply:
I'm not aware of the copy paste feature.is it OS specific? A terminal-specific?

Reply:
```suggestion
This variable lets you enable the mouse in Visual mode. Turning this variable on will be useful when you want to use your mouse to scroll through your disassembly.
```

What I was inferring is that, one will be able to use his or her mouse to select (drag and select) stuff on the Visual mode.

It's not a feature. I had used this variable just for that.
It's probably the best to not talk about it. I think, because the first sentence already covers this point.